packages/Theme/package.json

```diff
@@ -29,15 +29,16 @@
     "dooboolab"
   ],
   "devDependencies": {
+    "typescript": "^4.2.4",
     "@emotion/native": "^11.0.0",
     "@emotion/react": "^11.1.5",
-    "typescript": "^4.2.4"
+    "react-responsive": "^8.2.0"
```

Comment:
Shouldn't this go to `dependency` instead of `devDependencies`?

Reply:
@hyochan  Yeah. I fixed it. Thanks.